config/vufind/geofeatures.ini

```diff
@@ -40,10 +47,46 @@ basemap_attribution = "<a href=""https://wikimediafoundation.org/wiki/Maps_Terms
 ;basemap_attribution = "© <a href=""http://www.openstreetmap.org/copyright"">OpenStreetMap</a> contributors, © <a href=""https://carto.com/attribution"">CARTO</a>"
 
 [MapSelection]
-; Currently these configurations are set in the searches.ini file in the 
-; [MapSelection] section.
-
+; To enable this feature, uncomment the default_top_recommend[] = MapSelection
+; in the default recommendations section of searches.ini
+;
+; This defines the coordinates of a search region that will be highlighted when
+; the user clicks the "Geographic Search" link next to the VuFind search box.
+; This should ideally cover a large area of the map where most/all of your
+; geographic points are located. If your dataset is not concentrated in one
+; geographic area, it is advised that you pick a default area, and do not use
+; the entire extent of the map for searching (otherwise the search may be slow).
+;
+; default_coordinates:  The default coordinates specified below are in decimal
+;            degrees, and are ordered as WENS (west, east, north, south). Ranges of
+;            valid values are:; -180 to 180 (longitude) and -85 to 85 (latitude).
+;            Note, to search from and to the international date line, use
+;            west = -179 and east = -180.
+;
+; height:    Height in pixels of the map selection interface.
+default_coordinates = "-95, 30, 72, 15"
+height           = 320
 
 [MapTab]
-; Currently these configurations are set in the config.ini file in the 
-; Geographic Display section part of the [Content] section.
+; Configures the map tab display
+;
+; mapLabels:  leave empty, file:filename, or driver
+;      Leave it empty – no map labels will be displayed (default)
+;      file:filename - specify a file name after the colon for the
+;             coordinate/label lookup file. Coordinates in file must
+;             be specified as WENS.
+;      driver - Use the getCoordinateLabels method of the record driver to fetch labels;
+;               by default this relies on the long_lat_label field in Solr,
+;               but you can override the behavior with custom record driver code.
+;               The field must be the same length as the number of coordinate sets.
+;               Coordinates will be matched to labels on an ordered basis such that
+;               label[0] will be assigned for coordinate[0] and so forth.
+; displayCoords: true or false. Default is false.
+;                If displayCoords is true, then the coordinate values from
+;                coordinate field will be displayed before the map label in the label popup.
+; graticule: true or false. Default is false. If graticule is true a lat/long grid will be
+;                displayed on the map.
+mapLabels        = file:geosearch_test_lookup.txt
```

Comment:
I assume that we don't really want this setting turned on by default. Do you mind reviewing this file and commenting out anything that we don't actually want to make active (in case there are any other settings that are not intended for default production use)?

Reply:
Will do. Thanks